Cargo.toml

```diff
@@ -60,6 +60,19 @@ all-features = true
 beta = ["ironcore-search-helpers"]
 blocking = []
 
+# NOTE: ironoxide requires a TLS implementation. Choose one of the following
+
+# enable to use statically compiled openssl on supported OpenSSL platforms; use with 'default-features = false'
+tls-vendored-openssl = ["reqwest/native-tls-vendored"]
+
+# enable to use rustls-tls; use with 'default-features = false'
+tls-rustls = ["reqwest/rustls-tls"]
+
+# dynamically use the target platform's native TLS
+tls-default = ["reqwest/default-tls"]
+
+default = ["tls-default"]
```

Comment:
Is there a reason this wouldn't be `default = ["reqwest/default-tls"]`?

Reply:
No. They are equivalent. I wanted it to be clear that you need to choose one of the three `tls-` features, and once I defined it, I figured I might as well use it.